62.CAD/vertex_shader.hlsl

```diff
@@ -16,11 +18,8 @@ float2 BezierTangent(float2 p0, float2 p1, float2 p2, float t)
 
 // TODO[Przemek] Remove this function and use our builtin bezier functions
 float2 QuadraticBezier(float2 p0, float2 p1, float2 p2, float t)
-{   
-    float tt = t * t;
-    float oneMinusT = 1.0f - t;
-    float oneMinusTT = oneMinusT * oneMinusT;
-    return p0 * oneMinusTT + 2.0f * p1 * oneMinusT * t + p2 * tt;
+{
```

Comment:
I had left it as a todo for @Przemog1, be ready to solve some conflicts.
remove the todo at least too :D